src/main/java/ReadManager.java

```diff
@@ -0,0 +1,25 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class ReadManager {
+
+    private final ArrayList<String> list = new ArrayList<>();
+    private final File file;
+
+    public ReadManager(File file) {
+        this.file = file;
+    }
+
+    public ArrayList<String> readFile() {
+        try (Scanner fileReader = new Scanner(file)) {
+            while (fileReader.hasNextLine()) {
+                list.add(fileReader.nextLine());
+            }
+        } catch (FileNotFoundException e) {
+            System.out.println("File not found.");
+        }
```

Comment:
You can simplify it, please use `Files` from the `nio` package, see
https://docs.oracle.com/javase/8/docs/api/java/nio/file/Files.html#readAllLines-java.nio.file.Path-

Reply:
simplified